pylint/checkers/base.py

```diff
@@ -1215,6 +1269,30 @@ def open(self):
             for name_type in group.split(':'):
                 self._name_group[name_type] = 'group_%s' % (group,)
 
+        regexps, hints = self._create_naming_rules()
+        self._name_regexps = regexps
+        self._name_hints = hints
+
+    def _create_naming_rules(self):
+        regexps = {}
+        hints = {}
+        naming_style = NAMING_STYLES[self.config.naming_style]
+
+        for name_type in naming_style.known_node_types():
+            regexps[name_type] = naming_style.get_regex(name_type)
+            overriden_value_name = "%s_rgx" % (name_type,)
+            overriden_value = getattr(self.config, overriden_value_name, None)
+            if overriden_value is not None:
+                regexps[name_type] = overriden_value
+
+            hints[name_type] = regexps[name_type].pattern
```

Comment:
Yeah, that would be interesting. So basically instead of 

```Invalid name service``

we could have

``Invalid name service, use SERVICE instead``?